index.html

```diff
@@ -91,5 +91,195 @@ <h2 class="project-title">Tonic</h2>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="stacks">
+                        <span>HTML</span>
+                        <span>CSS</span>
+                        <span>Javascript</span>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="my-individual-project">
+            <div class="content-wrapper">
+                <div class="project-snaphot">
+                    <img src="./assets/orange-project.png" alt="porfolio-snapshot">
+                </div>
+                <div class="project-details">
+                    <h2 class="project-title">Tonic</h2>
+                    <div class="sub-title">
+                        <span>CANOPY</span>
+                        <img src="./assets/Counter.png" alt="counter">
+                        <span class="dev-title">Back End Dev</span>
+                        <img src="./assets/Counter.png" alt="counter">
+                        <span class="year-produced">2021</span>
+                    </div>
+                    <p class="project-description">
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="stacks">
+                        <span>HTML</span>
+                        <span>CSS</span>
+                        <span>Javascript</span>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="my-individual-project">
+            <div class="content-wrapper">
+                <div class="project-snaphot">
+                    <img src="./assets/colorful.png" alt="porfolio-snapshot">
+                </div>
+                <div class="project-details">
+                    <h2 class="project-title">Multi-Post Stories</h2>
+                    <div class="sub-title">
+                        <span>CANOPY</span>
+                        <img src="./assets/Counter.png" alt="counter">
+                        <span class="dev-title">Back End Dev</span>
+                        <img src="./assets/Counter.png" alt="counter">
+                        <span class="year-produced">2021</span>
+                    </div>
+                    <p class="project-description">
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="stacks">
+                        <span>HTML</span>
+                        <span>CSS</span>
+                        <span>Javascript</span>
+                    </div>
```

Comment:
- [ ] Great work implementing your technologies lists 💯. However, please kindly ensure to use `<ul>` and `<li>` semantic tags instead of `<div>` and `<span>` tags. 🤓 This will boost the readability and accessibility of your web. 🔧